contracts/scripts/DeployScripts.s.sol

```diff
@@ -25,6 +25,7 @@ contract DeployScript is Script {
             0xfA0B0f5d298d28EFE4d35641724141ef19C05684 // Placeholder for now, L1 preconf.eth address
         );
         uint16 feePercent = 2;
+        uint16 penaltyPercent = 5;
```

Comment:
We should make the feePercent and penaltyPercent vars more descriptive, `bidderFeePercent` and `providerFeePercent` or similar